package.json

```diff
@@ -10,8 +10,15 @@
     "preview": "vite preview"
```

Comment:
Why did you use Vite? What is it used for and what are the benefits in contrast to fx Webpack? 
Please be sure to elaborate on your report paper / exam.